db/re/item_db.txt

```diff
@@ -1781,7 +1781,7 @@
 2573,Archangel_Wing,Archangel Wings,4,0,,200,,18,,1,0xFFFFFFFF,63,2,4,,0,1,1,{},{},{}
 2574,Ur_Manteau,Ur's Manteau,4,20,,600,,20,,0,0x00000080,56,2,4,,100,1,0,{ bonus bMaxHPrate,2; bonus2 bSubEle,Ele_Neutral,10; },{},{}
 2575,Peuz_Manteau,Peuz's Manteau,4,20,,600,,16,,0,0x00000080,56,2,4,,100,1,0,{ bonus bFlee,10; bonus bCritical,10; },{},{}
-2576,Bravery_Bag,Adventurer's Backpack,4,0,,200,,20,,1,0xFFFFFFFF,63,2,4,,0,1,2,{ skill "BS_GREED",1; .@r = getrefine(); if(.@r>6) { if(readparam(bStr)>=90){ bonus bBaseAtk,20; } if(readparam(bInt)>=90){ bonus bMatk,30; } if(readparam(bVit)>=90){ bonus2 bSubEle,Ele_Neutral,10; } if(readparam(bAgi)>=90){ bonus bAspdRate,8; } if(readparam(bDex)>=90){ bonus bLongAtkRate,5; } if(readparam(bLuk)>=90){ bonus bCritAtkRate,10; } } if(.@r>8) { if(readparam(bStr)>=90){ bonus bBaseAtk,10; } if(readparam(bInt)>=90){ bonus bMatk,20; } if(readparam(bVit)>=90){ bonus2 bSubEle,Ele_Neutral,5; } if(readparam(bAgi)>=90){ bonus bAspd,1; } if(readparam(bDex)>=90){ bonus bLongAtkRate,5; } if(readparam(bLuk)>=90){ bonus bCritAtkRate,5; } } },{},{}
+2576,Bravery_Bag,Adventurer's Backpack,4,0,,200,,20,,1,0xFFFFFFFF,63,2,4,,0,1,2,{ skill "BS_GREED",1; .@r = getrefine(); .@str = readparam(bStr); if(.@str>=90){ if(.@r>=7) bonus bBaseAtk,(.@r>=9)?30:20; } .@int = readparam(bInt); if(.@int>=90){ if(.@r>=7) bonus bMatk,(.@r>=9)?50:30; } .@vit = readparam(bVit); if(.@vit>=90){ if(.@r>=7) bonus2 bSubEle,Ele_Neutral,(.@r>=9)?15:10; } .@agi = readparam(bAgi); if(.@agi>=90){ if(.@r>=7) bonus bAspdRate,8; if(.@r>=9) bonus bAspd,1; } .@dex = readparam(bDex); if(.@dex>=90){ if(.@r>=7) bonus bLongAtkRate,(.@r>=9)?10:5; } .@luk = readparam(bLuk); if(.@luk>=90){ if(.@r>=7) bonus bCritAtkRate,(.@r>=9)?15:10; } },{},{}
```

Comment:
I would rather write it
```
skill "BS_GREED",1;
.@r = getrefine();
if (.@r >= 7) {
	if (readparam(bStr) >= 90) { bonus bBaseAtk,(.@r>=9)?30:20; }
	if (readparam(bInt) >= 90) { bonus bMatk,(.@r>=9)?50:30; }
	if (readparam(bVit) >= 90) { bonus2 bSubEle,Ele_Neutral,(.@r>=9)?10:5; }
	if (readparam(bAgi) >= 90) { bonus bAspdRate,8; if (.@r>=9) { bonus bAspd,1; } }
	if (readparam(bDex) >= 90) { bonus bLongAtkRate,(.@r>=9)?10:5; }
	if (readparam(bLuk) >= 90) { bonus bCritAtkRate,(.@r>=9)?15:10; }
}
```

(note that our previous value `if(readparam(bVit)>=90){ bonus2 bSubEle,Ele_Neutral,10; }` for refine above +7 was wrong)

Reply:
You're right. xD